grails-app/views/report/showTransactionReport.gsp

```diff
@@ -112,7 +116,8 @@
 										name="tags"
 									    id="tags"
 									    value="${params?.tags}"
-									    multiple="true"
+										data-placeholder="${g.message(code: "default.selectSomeOptions.label", default: "Select some option")}"
```

Comment:
Change this to 
```
default.selectOptions.label=Select Options
```

I think the default "Select some option(s)" (or whatever) is provided by the chosen plugin and I'm not a fan.